src/CommunityPatch/Patches/PartyMoralePatch.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Linq;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.Core;
+using TaleWorlds.Localization;
+using HarmonyLib;
+using Helpers;
+using TaleWorlds.CampaignSystem.SandBox.GameComponents;
+using TaleWorlds.CampaignSystem.SandBox.GameComponents.Map;
+using TaleWorlds.CampaignSystem.SandBox.GameComponents.Party;
+using TaleWorlds.Library;
+using static CommunityPatch.HarmonyHelpers;
+
+namespace CommunityPatch.Patches {
+
+  internal class PartyMoralePatch : PatchBase<PartyMoralePatch> {
```

Comment:
A more specific class name might be good idea (Relating to Food Variety), can't imagine it's the only bug around party morale.

Reply:
`MaxFoodVarietyPartyMoralePatch` maybe?